R/capacity/health-inequalities/england/quality/nhs-trust-cqc-ratings.R

```diff
@@ -0,0 +1,185 @@
+# Load packages
+library(tidyverse)
+library(readODS)
+library(geographr)
+library(readxl)
+library(sf)
+
+source("R/utils.R") # for download_file()
+
+# Source page: https://www.cqc.org.uk/about-us/transparency/using-cqc-data#directory
+# Info on how is rated https://www.cqc.org.uk/what-we-do/how-we-do-our-job/how-we-rate-trusts-their-use-resources
+# Download the data
+tf <- download_file("https://www.cqc.org.uk/sites/default/files/01_December_2021_Latest_ratings.ods", "ods")
```

Comment:
This link is now unfortunately broken

Reply:
Fixed in https://github.com/britishredcrosssociety/resilience-index/pull/6/commits/2aae0a850b578b217a27a6b1c5312187027b1f07.